terraform/data_replication/variables.tf

```
@@ -124,6 +124,10 @@ locals {
     {
       name      = "RAILS_MASTER_KEY"
       valueFrom = var.rails_master_key_path
+    },
+    {
+      name      = "RO_DB_PASSWORD"
```

```suggestion
      name      = "READ_ONLY_DB_PASSWORD"
```

Is there any reason not to use the full name? It took me a while to work out what "ro" stood for reading this!